packages/ember-views/lib/mixins/view_support.js

```diff
@@ -620,7 +620,7 @@ export default Mixin.create({
     this._super(...arguments);
 
     if (!this.elementId && this.tagName !== '') {
-      this.elementId = guidFor(this);
```

Comment:
Sorry for the confusion – github attached the comment to one line above. I meant to say that we shouldn't have to do `this.id || guidFor(this);` at this point – because we should already have normalized it to `elementId` by now.